bin/population/pycbc_population_plots

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2017 Vaibhav Tiwari
+
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+"""
+This script reads the rates priors and posteriors and plots them
+"""
+
+__author__ = "Vaibhav Tiwari"
+__email__ = "vaibhav.tiwari@ligo.org"
+__version__ = "0.0"
+__date__ = "31.10.2017"
+
+import logging
+import argparse
+import pycbc.version
+
+import h5py
+import pylab
+import numpy as np
+import scipy.stats as ss
+from matplotlib.pyplot import cm
+from pycbc.population import rates_functions as rf
+from numpy import logaddexp, log, newaxis, expm1
+
+# Parse command line
+parser = argparse.ArgumentParser(description = 
+                                      "Plot rate prior and posteriors.")
+parser.add_argument('--prior-samples', dest='prior_samples', required = True, 
+                  help="Samples for rate prior.")
```

Comment:
suggest to change help to 'File containing rate prior samples'. 
Also you don't need to use 'dest' kwargs for all the arguments, they are automatically assigned. Makes the code simpler to remove them.

Reply:
changed description. Removed dest!